templates/single-product/add-to-cart/external.php

```diff
@@ -19,7 +19,7 @@
 
 do_action( 'woocommerce_before_add_to_cart_form' ); ?>
 
-<form class="cart" action="<?php echo esc_url( $product_url ); ?>" method="get">
+<form action="<?php echo esc_url( $product_url ); ?>" method="get">
```

Comment:
Removing the class shouldn't change anything. If you are not getting the correct behavior it seems likely to be a theme/plugin issue on your side, as I can't reproduce your problem on a default theme or any other theme I try.

You can see in the `form` action and method attributes that the form will make a GET request to the external product's URL when the form is submitted (i.e. Add-to-cart button is clicked).